cmd/event-notification.go

```diff
@@ -102,7 +104,15 @@ func (evnot *EventNotifier) InitBucketTargets(ctx context.Context, objAPI Object
 		return nil
 	}
 
-	logger.LogIf(ctx, evnot.targetList.Add(globalConfigTargetList.Targets()...))
+	if err := evnot.targetList.Add(globalConfigTargetList.Targets()...); err != nil {
+		return err
+	}
+
+	go func() {
+		for e := range evnot.eventsQueue {
```

Comment:
when do we send an event to eventsQueue?

Reply:
@vadmeste ^^

Reply:
> when do we send an event to eventsQueue?
Fixed, good catch